components/Statistic/pageHelpers.ts

```diff
@@ -0,0 +1,44 @@
+import { PostDocumentWithoutBody } from "interfaces";
+
+import { PostsByMonthType } from ".";
+
+export const counterMarginBottom = (count: number | undefined) => {
+  if (!count) {
+    return { mb: "55px" };
+  }
+
+  if (count === 1) {
+    return { mb: "26px" };
+  }
+
+  return {};
+};
+
+export const getPostsByYearAndMonth = (
+  year: number,
+  posts: PostDocumentWithoutBody[]
+) => {
+  const postsByMonth: PostsByMonthType = {};
+  const filteredPosts = posts.filter(
+    (post) => new Date(post.date).getFullYear() === year
+  );
+
+  filteredPosts.map((post) => {
```

Comment:
https://alex-code.com/post/2023-05-07-whats-the-difference-beetween-map-and-forEach-methods